drivers/bus/acpi_new/main.c

```diff
@@ -0,0 +1,16 @@
+#include "precomp.h"
+//#define NDEBUG
+#include <debug.h>
+
+CODE_SEG("INIT")
+NTSTATUS
+NTAPI
+DriverEntry (
+    PDRIVER_OBJECT  DriverObject,
+    PUNICODE_STRING RegistryPath
+    )
+{
+    DPRINT("ACPI_NEW: Entry %X\n",  ACPIInitUACPI());
```

Comment:
The call will be stripped from release builds. Is that intended?

Reply:
this was just to compile something, itll be stripped out eventually dont worry :)

Reply:
Just so we are all on the same page: the `ACPIInitUACPI()` function is not called in a release build.
Is that intentional?